SPECS/kernel/config_aarch64

```diff
@@ -2626,6 +2626,7 @@ CONFIG_ZRAM_DEF_COMP_LZORLE=y
 # CONFIG_ZRAM_DEF_COMP_LZO is not set
 CONFIG_ZRAM_DEF_COMP="lzo-rle"
 CONFIG_ZRAM_WRITEBACK=y
+CONFIG_ZRAM_TRACK_ENTRY_ACTIME=y
```

Comment:
Same as in `config`
Result of https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/b7c3fd65a3eaa7a7639ddec8b817a824e814bf37

This is a new option seperates out ZRAM access time tracking or "ac-time tracking" which was previously a part of the memory tracking option. It is selected if ZRAM_MEMORY_TRACKING is enabled which is on in ARM64 but not in x86_64

Reply:
Keeping on to stay inline with what was previously present for aarch64